src/OpenTK/MouseCursor.cs

```diff
@@ -28,28 +28,55 @@
 #endregion
 
 using System;
+using System.Collections;
+using System.Collections.Generic;
 
 namespace OpenTK
 {
     /// <summary>
+    /// Stores information about frame timing
+    /// </summary>
+    public struct MouseCursorFrameInformation
```

Comment:
Because we're dealing with animation, a better name rather than information might be a 'keyframe' or 'key'.